client/components/LearnerViewDirective.js

```diff
@@ -568,6 +586,33 @@ tie.directive('learnerView', [function() {
           }
         };
 
+        $scope.changeQuestionSet = function() {
```

Comment:
This is also consistent with the implementation of the themes -- should we change the way we do theming, as well?